packages/control-property-editor-common/src/api.ts

```diff
@@ -51,11 +59,17 @@ export const SCENARIO = {
     UiAdaptation: 'UI_ADAPTATION'
 } as const;
 
+export enum PropertyType {
+    Configuration = 'configuration',
+    ControlProperty = 'controlProperty'
+}
+
 export type Scenario = (typeof SCENARIO)[keyof typeof SCENARIO];
 
 interface ControlPropertyBase<T, V, E> {
     type: T;
     editor: E;
+    propertyType: PropertyType;
```

Comment:
The reason was to simplify differentiating Control Property and Configuration Property when creating respective changes.
https://github.com/SAP/open-ux-tools/blob/e71281eef863ea3740854a5e5a2ec9fb994fa767/packages/preview-middleware-client/src/cpe/changes/flex-change.ts#L37-L38

if i use the line 38 for branching the challenges are 
1. the table control have properties with the same name as configuration properties
2. More checks would be required to see if these properties are ignored by designtime metadata in case of control property

Because of this i needed to pass this down as well from the UI like in line 81
https://github.com/SAP/open-ux-tools/blob/e71281eef863ea3740854a5e5a2ec9fb994fa767/packages/control-property-editor/src/panels/properties/StringEditor.tsx#L77-L83

This could be refactored when we cache properties in outline node what do you think?